layers/+misc/xah-fly-keys/packages.el

```diff
@@ -0,0 +1,49 @@
+;;; packages.el --- xah-fly-keys layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Grant Shangreaux <shshoshin@protonmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Code:
+
+(defconst xah-fly-keys-packages
+  '(
+    xah-fly-keys
+    magit
+    )
+  )
+
+
+(defun xah-fly-keys/init-xah-fly-keys ()
+  (use-package xah-fly-keys
+    ;; :defer t
```

Comment:
I'm not sure that `xah-fly-keys` has autoloads set up.  If i tried defering this, I was having issues getting the mode to be active on startup.